src/graphql/resolvers/custom/patient.js

```diff
@@ -61,6 +77,7 @@ module.exports = {
         updateGeneralPractitioner:
         // eslint-disable-next-line no-unused-vars
             async (parent, args, context, info) => {
+                const deletePractitioner = args.remove;
```

Comment:
we tend to use JSDoc to define types of variables: https://jsdoc.app/about-getting-started.html